io.itch.Itch.yaml

```diff
@@ -0,0 +1,143 @@
+app-id: io.itch.Itch
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.winehq.Wine
+base-version: stable-22.08
+command: itch-run
+separate-locales: false
+finish-args:
+  - --allow=devel # For Wine crash handling
+  - --allow=multiarch
+  
+  - --device=all
+  
+  - --share=ipc
+  - --share=network
+
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.ScreenSaver
+
+  - --env=PATH=/app/bin:/usr/bin:/app/utils/gamescope/bin
+  - --env=WINEPREFIX=/var/data/wine
+  - --env=WINEDLLOVERRIDES=mscoree,mshtml='
+  - --env=WINEDLLPATH=/app/dlls/lib32:/app/dlls/lib:/app/lib32/wine/wined3d:/app/lib/wine/wined3d
+  - --env=LD_LIBRARY_PATH=/app/lib:/app/lib32
+  - --env=GST_PLUGIN_SYSTEM_PATH=/app/lib/gstreamer-1.0:/usr/lib/x86_64-linux-gnu/gstreamer-1.0:/app/lib32/gstreamer-1.0:/usr/lib/i386-linux-gnu/gstreamer-1.0
+  
+  - --persist=.wine
+  - --persist=.wine64
+#  - --persist=.
+
+inherit-extensions:
+  - org.freedesktop.Platform.GL32
+  - org.freedesktop.Platform.ffmpeg-full
+  - org.freedesktop.Platform.Compat.i386
+  - org.freedesktop.Platform.ffmpeg_full.i386
+  - org.winehq.Wine.gecko
+  - org.winehq.Wine.mono
+  - org.winehq.Wine.DLLs
+
+add-extensions:
+
+  org.freedesktop.Platform.Compat.i386.Debug:
+    directory: lib/debug/lib/i386-linux-gnu
+    no-autodownload: true
+    version: "22.08"
+
+  com.valvesoftware.Steam.Utility.gamescope:
+    add-ld-path: lib
+    directory: utils/gamescope
+
+modules:
+
+  - name: zypak
+    sources:
+      - type: git
+        url: https://github.com/refi64/zypak
+        tag: v2022.04
+        commit: 55a60d110301e8dd37b631503c3524ab7baaf7aa
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\d.]+)$
+          
+  - name: itch-resources
+    buildsystem: simple
+    build-commands:
```

Comment:
all of these can go in the post-install of `itch`.

Reply:
Can post-install have their own sources? As currently `itch` downloads -> moves all content from the archive, which I want to keep as isolated from change as possible.

Reply:
it's a `buildsystem simple` so yuo can do anything in the build-commands. Actually you don't need post-install for this kind of buildsystem.